### ISSUE TITLE
"Cancel" option in UI for history

### ISSUE THREAD

**Issue by slapphappyjazz** [tobymao/18xx#6254]
When you touch the game log, it promps two buttons - "Undo to here" and "Review from here".  It would be useful to have a "Cancel" button as well, to handle the case where I'm scrolling back through the log to read other people's moves and I accidentally touch the log itself.
I know you can just scroll back to the end of the log but a "cancel" button would be simpler and wouldn't require scrolling away from the area I'm looking at. IMO it would also be more intuitive.  I assume this would be quite simple to implement, too.

**Comment by scottredracecar** [comment]
Clicking the log again cancels it. I find this to be an easily discoverable action. Feel free to reopen if I am misunderstanding.

**Comment by slapphappyjazz** [comment]
So I tried this again at the weekend, using an ipad, and I found that touching the log again would sometimes just open the pop-up at a new location, and sometimes would cancel it.  It seemed sensitive to where you click on the log and there wasn't a visual cue.  I assume that if you click close enough to the current location, that cancels the popup, whereas if you touch it at a place it recognises as a new location, that moves the popup, but the lack of a visual cue makes it seem a bit random.
 I do think a "cancel" buttom would be easier, although I understand it's not a high priority.
